src/lib/OpenCvProvider.js

```diff
@@ -6,53 +6,39 @@ const { Consumer: OpenCvConsumer, Provider } = OpenCvContext
 
 export { OpenCvConsumer, OpenCvContext }
 
-const scriptId = 'opencv-react'
-const moduleConfig = {
-  wasmBinaryFile: 'opencv_js.wasm',
-  usingWasm: true
-}
-
-export const OpenCvProvider = ({ openCvPath, children, onLoad }) => {
-  const [loaded, setLoaded] = React.useState(false)
-
-  const handleOnLoad = React.useCallback(() => {
-    if (onLoad) {
-      onLoad(window.cv)
-    }
-    setLoaded(true)
-  }, [])
+export const OpenCvProvider = ({ openCvVersion = '4.5.5', openCvPath = '', children }) => {
+  const [cvInstance, setCvInstance] = React.useState({ loaded: false, cv: undefined });
 
   React.useEffect(() => {
+    const scriptId = 'opencv-react'
```

Comment:
this can be left outside where it was, no need to redefine it at each effect run

Reply:
agreed.